bin/load.js

```diff
@@ -25,6 +25,7 @@ const refseq = require('../src/refseq');
 const PMC4468049 = require('../src/PMC4468049');
 const PMC4232638 = require('../src/PMC4232638');
 const uberon = require('../src/uberon');
+const variants = require('../src/variants');
```

Comment:
might be clearer to name the file and/or const 'nonspecific-variants'?

Reply:
The name has to match was the user users for the input argument. I went with "variants" because "nonSpecificVariants" would be a bit long to type as an input argument. This is the only variants-only loader at the moment so it probably ok to have the short name i think?